e2e/rbd.go

```diff
@@ -805,14 +805,14 @@ var _ = Describe("RBD", func() {
 				}
 				scOpts := map[string]string{
 					"encrypted":       "true",
-					"encryptionKMSID": "secrets-metadata-test",
+					"encryptionKMSID": "vault-test",
```

Comment:
Do we still have an other test that uses `"encryptionKMSID": "secrets-metadata-test"` ?

Reply:
No we don't have another test for pvc-pvc clone.

Reply:
In that case, you should probably copy this test, and create a 2nd one. We need to keep testing the `secrets-metadata` KMS too.

Reply:
okay

Reply:
@nixpanic In this case we can't use validateEncryption as kms != "", because we have encryption as true and kms as  "". Either we can use validateEncrypton or in place of vault we can pass secret-metadata as kms value. Please suggest.

Reply:
Feel free to pass `secret-metadata` or similar. And maybe define `const noKMS = ""` so that you can use that instead of passing and checking for `""` to make it more readable.